pkgs/development/python-modules/rauth/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, requests
+, pytestCheckHook
+, mock
+, coverage
+, flake8
+, nose
+, pycrypto
+, yanc
+,
+}:
+buildPythonPackage rec {
+  pname = "rauth";
+  version = "0.7.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "litl";
+    repo = "rauth";
+    rev = version;
+    hash = "sha256-wRKZbxZCEfihOaJM8sk8438LE++KJWxdOGImpL1gHa4=";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "fix-pycrypdodome-replacement-for-pycrypto.patch";
+      url = "https://github.com/litl/rauth/commit/7fb3b7bf1a1869a52cf59ee3eb607d318e97265c.patch";
+      sha256 = "sha256-jiAIw+VQ2d/bkm2brqfY1RUrNGf+lsMPnoI91gGUS6o=";
+    })
+  ];
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  doCheck = true;
+
+  pythonImportsCheck = [ "rauth" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mock
+    coverage
+    flake8
+    nose
+    pycrypto
+    yanc
```

Comment:
The test will pass without anything but pycrypto, I guess.

Reply:
`mock` & `nose` where in fact [still necessary](https://github.com/NixOS/nixpkgs/compare/bb60250fa9142e184d9893b23520adb1d0cc01fe..27344c890fe0e4462daaf9959ca22aede8f4c771)